client/logmon/logmon.go

```diff
@@ -216,7 +213,6 @@ func newLogRotatorWrapper(path string, logger hclog.Logger, rotator io.WriteClos
 	}
 
 	if err != nil {
-		logger.Error("Failed to create FIFO", "stat_error", serr, "create_err", err)
```

Comment:
I'd like to keep this log line, because it would've helped us figure out the golang regression much earlier and could be useful for debugging weird issues in the future.